Classes/Provider/BackendUserProvider.php

```diff
@@ -28,20 +30,44 @@ public function get(array $data)
     {
 
         $this->objectManager = GeneralUtility::makeInstance('TYPO3\\CMS\\Extbase\\Object\\ObjectManager');
-        $this->backendUserRepository = $this->objectManager->get('TYPO3\\CMS\\Extbase\\Domain\\Repository\\BackendUserRepository');
+        $this->backendUserRepository = $this->objectManager->get('TYPO3\\CMS\\Beuser\\Domain\\Repository\\BackendUserRepository');
 
         /** @var BackendUser[] $backendUsers */
         $backendUsers = $this->backendUserRepository->findAll();
         foreach($backendUsers as $backendUser)
         {
-            $data['users']['backend'][] = array(
+            $backendUserData = array(
                 'userName' => $backendUser->getUserName(),
                 'realName' => $backendUser->getRealName(),
+                'emailAddress' => $backendUser->getEmail(),
                 'description' => $backendUser->getDescription(),
-                'lastLogin' => $backendUser->getLastLoginDateAndTime()
+                'lastLogin' => $backendUser->getLastLoginDateAndTime(),
+                'avatar' => ''
             );
+            if (GeneralUtility::compat_version('7.5')) {
```

Comment:
please remove the avatar, I don't see any need for that and it requires a lot of code to get all this info in an eID call which should be fast